doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,95 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+ 
+* :py:class:`~._signals.hologram_image.HologramImage`
+* :py:class:`~._signals.wave_image.ComplexSignal2D`
```

Comment:
This class is not specifically for electron holography, so you should remove it here (change the text around it accordingly).